tests/conftest.py

```diff
@@ -97,3 +97,7 @@ def second_shipment(mocked_engine_rpc, mocked_iot_api):
                                    shipper_wallet_id=SHIPPER_ID,
                                    storage_credentials_id=random_id(),
                                    owner_id=USER_ID)
+
+@pytest.fixture
+def entity_shipment_relationship(json_asserter, shipment):
+    return json_asserter.EntityRef(resource='Shipment', pk=shipment.id)
```

Comment:
I'm not sure this needs to be a fixture, I think it sacrifices code readability without really DRYing much of anything.